sdk/textanalytics/Azure.AI.TextAnalytics/ToStringGuidence.md

```diff
@@ -0,0 +1,30 @@
+# ToString
```

Comment:
could something be added to codegen? have a list of properties?
is it worth it though having ToString as a simple thing to implement...